apps/server/src/setup/loadDb.ts

```diff
@@ -44,35 +47,49 @@ const populateDb = (directory: string, filename: string): string => {
 };
 
 /**
- * @description parses a json file to the adapter
- * It will create an empty file from the model if the parsing fails
+ * Handles a corrupted fle by copying it to a corrupted folder
+ * Eventual recovered data will be added to a new file
  */
-const parseDatabase = async (fileToRead: string, adapterToUse: Low<DatabaseModel>) => {
+async function handleCorruptedDb(filePath: string) {
   try {
-    // this will throw if file is not valid
-    parseProjectFile(fileToRead);
-    await adapterToUse.read();
-  } catch (error) {
-    adapterToUse.data = dbModel;
+    const fileName = getFileNameFromPath(filePath);
+    const newFilePath = join(resolveCorruptedFilesDirectory, fileName);
+
+    ensureDirectory(resolveCorruptedFilesDirectory);
+    copyFileSync(filePath, newFilePath);
```

Comment:
Should we maybe move the broken file or delete it after copy so it doesn't stay in the list?

Reply:
We basically continue working on the "broken" file.
We rename it to have the (recovered) append and overwrite its contents